packages/tables/src/Concerns/CanReorderRecords.php

```diff
@@ -36,14 +36,15 @@ public function reorderTable(array $order): void
 
             $model = app($this->getTable()->getModel());
             $modelKeyName = $model->getKeyName();
+            $wrappedModelKeyName = DB::connection()->getSchemaGrammar()->wrap($modelKeyName);
```

Comment:
```suggestion
            $wrappedModelKeyName = $model->getConnection()->getSchemaGrammar()->wrap($modelKeyName);
```

in case the model has a customized `$connection` set, otherwise the default is used